### PR TITLE
Add backend app tags to relevant frontend tests

### PR DESCRIPTION
**Pull request by KludgeKML**
- We want to make sure that when these apps are deployed frontend can still access them.

### CODE REVIEW

File: tests/frontend.spec.js [hunk 1]
```diff
@@ -120,15 +120,15 @@ test.describe("Frontend", { tag: ["@app-frontend", "@domain-www"] }, () => {
...
+  test("local transaction page", { tag: ["@app-local-links-manager", "@locations-api"] }, async ({ page }) => {
     await page.goto("/pay-council-tax");
     await expect(page.getByRole("heading", { name: "Pay your Council Tax" })).toBeVisible();
     await page.getByLabel("Enter a postcode").fill("WC2B 6NH");
     await page.getByRole("button", { name: "Find" }).click();
     await expect(page.getByText("London Borough of Camden", { exact: true })).toBeVisible();
   });
 
-  test("place page", async ({ page }) => {
+  test("place page",{ tag: ["@app-places-manager-manager", "@locations-api"] }, async ({ page }) => {
```
Comment (theseanything) [inline tests/frontend.spec.js:131]:
```suggestion
  test("place page",{ tag: ["@app-places-manager", "@locations-api"] }, async ({ page }) => {
```

File: tests/frontend.spec.js [hunk 2]
```diff
@@ -66,7 +66,7 @@ test.describe("Frontend", { tag: ["@app-frontend", "@domain-www"] }, () => {
     await expect(page.getByText("Electoral Services Dudley")).toBeVisible();
   });
 
-  test("find your local council", async ({ page }) => {
+  test("find your local council", { tag: ["@app-local-links-manager", "@locations-api"] }, async ({ page }) => {
```
Comment (theseanything) [inline tests/frontend.spec.js:69]: Did you mean
```suggestion
  test("find your local council", { tag: ["@app-local-links-manager", "@app-locations-api"] }, async ({ page }) => {
```

File: tests/frontend.spec.js [hunk 3]
```diff
@@ -120,15 +120,15 @@ test.describe("Frontend", { tag: ["@app-frontend", "@domain-www"] }, () => {
     await expect(page.getByRole("button", { name: "Start now" })).toBeVisible();
   });
 
-  test("local transaction page", async ({ page }) => {
+  test("local transaction page", { tag: ["@app-local-links-manager", "@locations-api"] }, async ({ page }) => {
```
Comment (theseanything) [inline tests/frontend.spec.js:123]:
```suggestion
  test("local transaction page", { tag: ["@app-local-links-manager", "@app-locations-api"] }, async ({ page }) => {
```

**Review by theseanything — APPROVED** [review]

**Comment by KludgeKML** [comment]
@theseanything thanks! (Have fixed up the commit with the suggestions)

**Pull request merged**